commander3/src/comm_tod_gain_mod.f90

```diff
@@ -659,7 +706,7 @@ subroutine get_pid_ranges(pid_ranges, tod, dgains, dipole_mods, window_sizes)
             jump_percentile = 0.995
          ! Currently, the 70-GHz ds
          case default
-            slow_smooth_window_size = 150
+            slow_smooth_window_size = 15
```

Comment:
I bet this should be a parameter, or at least be hardcoded per experiment in the constructor

Reply:
Actually, this whole parameter selection method is a bit of a disaster, we should probably re-think it

Reply:
I think @eirikgje was going to take a look at the gain module in general.

Reply:
We'll fix that later, when all is merged in. This branch isn't the right place to fix it.